.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.18-openshift-4.12
+  tag: rhel-8-release-golang-1.18-openshift-4.13
```

Comment:
I think the image is `rhel-8-release-golang-1.19-openshift-4.13`

also please update the Dockerfiles with the rhel7 postfix

Reply:
Bumping golang is not in the scope of this PR. This is just for aligning the operator config to what ART expects. Same applies for Dockerfile changes